pkg/kubelet/util/csr/csr.go

```diff
@@ -39,7 +49,7 @@ import (
 func RequestNodeCertificate(client certificatesclient.CertificateSigningRequestInterface, privateKeyData []byte, nodeName types.NodeName) (certData []byte, err error) {
```

Comment:
@jcbsmpsn is this function used for client cert rotation? If so, this will reuse an existing CSR and not request a new cert. if we're using this from the rotation code, we need to be able to pass in an optional value to add to the hash to produce a new stable CSR name (I'd probably pass in the serial number or expiration date or some attribute of the current certificate)